docs/faq-troubleshooting.md

```diff
@@ -0,0 +1,153 @@
+suggest a new feature, if you need assistance with a contribution, or if you
+found any bug.
+* [Visit the source{d} Forum](https://forum.sourced.tech) where users and community
+members discuss anything source{d} related. You will find there some common questions
+from other source{d} users, or ask yours.
+* [join our community on Slack](https://sourced-community.slack.com/join/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM),
+and talk with some of our engineers.
+
+
+## How Can I Update My Version Of source{d} CE?
+
+When there is a new release of **source{d} CE**, it is noticed every time a `sourced`
+command is called. When it happens you can download the new version from
+[src-d/sourced-ce/releases/latest](https://github.com/src-d/sourced-ce/releases/latest),
+and proceed as it follows:
+
+(You can also follow these steps if you want to update to any beta version, to
+downgrade, or to use your own built version of **source{d} CE**)
+
+1. download the new binary for your OS
+1. move the binary to the right place ([see Quickstart. Install](quickstart/2-install-sourced.md))
+1. run `sourced compose download`
+1. run `sourced restart` to apply the new configuration.
+
+This process will reinstall **source{d} CE** with the new components, but it will
+keep your data (repositories, charts, dashboards, etc). If you want to replace
+all the current dashboards with the ones from the release that you just installed,
+you have two alternatives:
+
+- run `sourced prune --all` before running `sourced init`; but if you were using
+repositories downloaded from a GitHub organization, they will be deleted, and
+downloaded again.
+- drop only the dashboards, and load the new ones, following also this other
+instructions: [How To Restore Dashboards and Charts to Defaults](#how-to-restore-dashboards-and-charts-to-defaults)
+
+
+## How To Restore Dashboards and Charts to Defaults
+
+In some circumstances, you may want to restore the UI dashboard and charts to its
+defaults. Following these steps, all the changes that you did in charts and
+dashboards will be replaced by the default ones for the version of **source{d} CE**
+that you're currently using. If you're using repositories from a GitHub organization,
+all its data will be preserved, and only charts and dashboards will be restarted.
+
+To do so, you only need to delete the docker volume containing the PostgreSQL
+database, and restart **source{d} CE**. It can be doing following these steps if
+you already have [Docker Compose](https://docs.docker.com/compose/) installed:
+
+```shell
+$ cd ~/.sourced/workdirs/__active__
+$ source .env
+$ docker-compose stop postgres
+$ docker-compose rm -f postgres
+$ ENV_PREFIX=`awk '{print tolower($0)}' <<< ${COMPOSE_PROJECT_NAME}`
+$ docker volume rm ${ENV_PREFIX}_postgres
+$ sourced restart
+```
+
+
+## How Can I See Logs of Running Components?
+
+```shell
+$ cd ~/.sourced/workdirs/__active__
```

Comment:
this could be replaced by
```shell
sourced logs [--follow] [component...]
```
if https://github.com/src-d/sourced-ce/pull/123 is merged